src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -727,10 +729,18 @@ function addTabStopsToScrollableElements() {
   const updateTabStops = () => {
     document
       .querySelectorAll(
-        "pre, " + // code blocks
-          ".nboutput > .output_area, " + // NBSphinx notebook output
-          ".cell_output > .output, " + // Myst-NB
-          ".jp-RenderedHTMLCommon", // ipywidgets
+        [
+          // code blocks
+          "pre",
+          // NBSphinx notebook output
+          ".nboutput > .output_area",
+          // Myst-NB
+          ".cell_output > .output",
+          // ipywidgets
+          ".jp-RenderedHTMLCommon",
+          // [rST table nodes](https://www.docutils.org/docs/ref/doctree.html#table)
+          ".pst-scrollable-table-container",
+        ].join(", "),
```

Comment:
Reformatted this section because when I added the new selector, `.pst-scrollable-table-container`, I got it wrong the first time.